cegui/src/XMLParserModules/TinyXML/XMLParser.cpp

```diff
@@ -64,45 +64,30 @@ namespace CEGUI
         d_handler = &handler;
 
         // Create a buffer with extra bytes for a newline and a terminating null
-        size_t size = source.getSize();
-        char* buf = new char[size + 2];
-        memcpy(buf, source.getDataPtr(), size);
```

Comment:
Not sure about performance difference of raw memory vs STL container in this case.
But it did make the code more readable.
I think accepting this change should be in the question of project's coding convention.

Reply:
Using the `std::string` later removes the need for a try/catch block later on.